.gitignore

```diff
@@ -1,5 +1,4 @@
 *~
-*.iml
```

Comment:
The gitignore plugin in IDEA told me that .iml could go if we had .idea, but it didn't say anything about .iws or .ipr. .iws/.ipr seems to be used for "file based projects", so it might not be covered by ".idea".